api/configure.md

```diff
@@ -3,4 +3,4 @@ uid: configure
 title: Configuration Operators
 ---
 
-Configuration operators belong in a top-level chain of operators between [core operators](xref:core) ([`CreateContext`](xref:OpenEphys.Onix1.CreateContext) and [`StartAcquisition`](xref:OpenEphys.Onix1.StartAcquisition)) to configure ONIX hardware.
\ No newline at end of file
+Device configuration operators belong in a top-level chain of operators between [`CreateContext`](xref:OpenEphys.Onix1.CreateContext) and [`StartAcquisition`](xref:OpenEphys.Onix1.StartAcquisition) to configure devices contained by ONIX hardware.
```

Comment:
Consider using the term hubs for clarify "to configure devices contained by ONIX hardware hubs."

Reply:
great idea!